scripts/commands/update-translations.js

```diff
@@ -52,11 +52,7 @@ export default async function updateTranslations() {
 			`wp i18n make-pot . ${ POT_FILE_PATH } --ignore-domain`
 		);
 
-		if ( makePotResult.stderr ) {
```

Comment:
I would love it if we can come up with some way to tell if the `wp i18n make-pot` command succeeded. Can we check the return code of the command or verify the output or something?

Reply:
Here is the output of the command the code update
![code update](https://github.com/Automattic/wc-calypso-bridge/assets/31164683/8df29576-9984-4c50-9edd-3d41edb00db4)

Reply:
Rather than always displaying that output, what if we only output it in the case where the `execPromise` has failed and we've fallen into the exception handler?

Something like this (untested)
```
} catch ( err ) {
    if ( makePotResult && makePotResult.stderr ) {
      warning( makePotResult.stderr );
    }
    return error( 'Error executing the script: ' + err.message );
}
```

Reply:
If `execPromise` has failed, then `makePotResult` doesn't exist. This can achieve similar:
```
} catch ( err ) {
	if ( err.stderr ) {
		warning( err.stderr )
	}
	return error( 'Error executing the script: ' + err.message );
}
```
Using a non-existent directory in the `make-pot` command produces the following result:

![Screenshot 2023-07-21 at 1 01 21 PM](https://github.com/Automattic/wc-calypso-bridge/assets/31164683/17dc6861-1edf-42e9-b9fe-d96f11eced8d)